elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
+    """
+    Peristimulus Time Histogram (PSTH) of a list of :attr:`neo.SpikeTrain`
```

Comment:
The word "stimulus" appears only in the title, and nowhere else in the docstring or the list of arguments. For greater clarity, I wonder if we shouldn't rename "t_start" to something like "onset", "stimulus_onset", "event_time", etc., and replace "t_stop" with "duration". I also think there is a case for making "t_start/onset/whatever" a required argument.